test/hotspot/jtreg/testlibrary/ctw/src/sun/hotspot/tools/ctw/CtwRunner.java

```diff
@@ -306,6 +306,13 @@ private String[] cmd(long classStart, long classStop) {
                 // StressSeed is uint
                 "-XX:StressSeed=" + rng.nextInt(Integer.MAX_VALUE)));
 
+        // Use this stress mode 10% of the time as it could make some long-running compilations likely to abort.
```

Comment:
I apply this stress mode randomly as it could hypothetically hide an infinite compilation bug if used unconditionally.
If it is later used in another test suite we might need similar randomization.